run.py

```diff
@@ -176,10 +176,8 @@ def install_py_libs():
 
   exclude_ext = ['.pth', '.pyc', '.egg-info', '.dist-info', '.so']
   exclude_prefix = ['setuptools-', 'pip-', 'Pillow-']
-  exclude = [
-    'test', 'tests', 'pip', 'setuptools', '_markerlib', 'PIL',
-    'easy_install.py', 'pkg_resources', 'pkg_resources.py'
-  ]
+  exclude = ['test', 'tests', 'pip', 'setuptools', '_markerlib', 'PIL',
+             'easy_install.py']
```

Comment:
Put them in one line.. and no need to squash/rebase.. I'm squashing all the PRs anyway.

Reply:
ping

Reply:
Ah, sorry... was busy with other stuff last month; will try to address it today.